### PR TITLE
SmartSyncExplorer for React Native

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

**Comment by wmathurin** [comment]
Separate repo for now. We could bring it into the main repo.

File: js/SearchBar.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+'use strict';
```
Comment (kchitalia) [inline js/SearchBar.js:27]: Please add Copyright info.

**Comment by bhariharan** [comment]
:+1: 

Should we pull this into the main repo?

**Comment by wmathurin** [comment]
I should probably move it to the main repo.
But at DF, I'll probably demo from the separate repo since it looks exactly like what you would get by running forceios.

**Pull request merged**